docs/HOWTO-run_tests.md

```diff
@@ -14,3 +14,21 @@ GRANT ALL PRIVILEGES ON DATABASE spoke_test TO spoke_test;
 ## SQLite Testing (simpler)
 
 1) Run `npm run test-sqlite`
+
+## End-To-End (Interactive Browser) Testing
+
+1. Remember to set `NODE_ENV=dev` 
+1. Setup DB
```

Comment:
When you mean setup DB, you mean like whats apart of the startup guide? Anyway we can be clearer here

Reply:
At the time of this PR I still wasn't sure what "SetupDB" meant. I've added a note to myself to elaborate.